builtins/util-xe.m4

```diff
@@ -4160,723 +4160,6 @@ declare void @__pseudo_prefetch_write_varying_3(<WIDTH x i64>, <WIDTH x MASK>) n
 declare void
 @__pseudo_prefetch_write_varying_3_native(i8 *, i32, <WIDTH x i32>,
```

Comment:
Please double check this file. Looks like some pseudo_gathers/pseudo_scatters declarations were not removed from it.

Reply:
Done